editor/editor/mode/visual.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import InserterButton from '../../inserter/button';
+
+const Blocks = ( { blocks, onChange } ) => {
+	const onChangeBlock = ( index ) => ( changes ) => {
+		const newBlock = {
+			...blocks[ index ],
+			changes
+		};
+
+		onChange( [
+			...blocks.slice( 0, index ),
+			newBlock,
+			...blocks.slice( index + 1 )
+		] );
+	};
+
+	return (
+		<div className="editor-mode-visual">
+			<div contentEditable>
```

Comment:
`contentEditable` was removed in #330 and we'll probably want to do the same here.